tools/echo_server/readme.md

```diff
@@ -47,15 +51,23 @@ The JSON file contains the following options:
     "verbose": false,
     "logging": false,
     "secure-connection": false,
+    "repeat-mode": false,
+    "repeat-interval-seconds": 1,
     "server-port": "9000",
     "server-certificate-location": "./certs/server.pem",
-    "server-key-location": "./certs/server.pem"
+    "server-key-location": "./certs/server.key"
 }
 ```
 # Running the Echo Server From the Command Line
 `go run echo_server.go`
 ## With a Custom Config Location
 `go run echo_server.go -config={config_file_path}`
+## With Repeat Mode on and a repeat interval of 3 seconds
+`go run echo_server.go -repeat=true -interval=3`
+## With Server Certificate and Key passed from command line args
+`go run echo_server.go -cert='<x509-cert-content>' -key='<x509-key-content>'`
```

Comment:
This breaks the current config method, so will need to update the docs accordingly.

Reply:
I will make this very clear in next rev:

Revised config file options:
```
"server-certificate": "<cert-content>",
"server-key": "<key-content>",
"server-certificate-location": "./certs/server.pem",
"server-key-location": "./certs/server.key"
```

New cmdline args:
`-cert` and `-key` 

Priority:
`-cert` > `server-certificate` in config file > `server-certificate-location` in config file
`-key` > `server-key` in config file > `server-key-location` in config file